simplq/src/components/pages/Admin/index.jsx

```diff
@@ -36,23 +39,22 @@ export default (props) => {
     return () => clearTimeout(timeoutId);
   }, [update]);
 
-  const addNewToken = (name, contactNumber) => {
-    return TokenService.create(name, contactNumber, false, queueId)
-      .then((response) => {
-        setTokens([
-          ...tokens,
-          {
-            tokenId: response.tokenId,
-            name,
-            contactNumber,
-            notifiable: false,
-            tokenStatus: response.tokenStatus,
-          },
-        ]);
-      })
-      .catch((err) => {
-        handleApiErrors(err);
-      });
+  const addNewToken = async (name, contactNumber) => {
+    try {
+      const response = await TokenService.create(name, contactNumber, false, queueId);
+      setTokens([
+        ...tokens,
+        {
+          tokenId: response.tokenId,
+          name,
+          contactNumber,
+          notifiable: false,
+          tokenStatus: response.tokenStatus,
+        },
+      ]);
+    } catch (err) {
+      handleApiErrors(err);
+    }
   };
```

Comment:
Just an aesthetic change. I don't think there will be any side effects, but do check @daltonfury42 .

Reply:
looks good